adafruit_epd/epd.py

```diff
@@ -176,7 +195,7 @@ def command(self, cmd, data=None, end=True):
 
         return ret
 
-    def _spi_transfer(self, data):
+    def _spi_transfer(self, data: Union[int, bytearray]) -> int:
```

Comment:
I think the return type here should be `Optional[int]` since it can return None sometimes instead of a number value.